src/node_native_module.cc

```diff
@@ -174,6 +174,48 @@ MaybeLocal<Function> NativeModuleLoader::CompileAsModule(
   return LookupAndCompile(context, id, &parameters, result);
 }
 
+Local<String> NativeModuleLoader::LoadInternalModuleSource(Isolate* isolate,
+                                                           const char* id) {
+#ifdef NODE_LOAD_INTERNAL_MODULES_FROM_DISK
+    // Filename on disk
```

Comment:
It looks like the indentation of this entire function is off? Does the linter complain?